client/src/components/QuestionsAndAnswers/QuestionAnswer.jsx

```diff
@@ -33,51 +37,49 @@ export default class QuestionAnswer extends React.Component {
             data.results.sort((a, b) => {
                 return b.question_helpfulness - a.question_helpfulness
             });
-            // add expanded to show 2 or more answers
             const qas = data.results.map((qa) => ({ ...qa, expanded: false }))
             this.setState({ qas })
         });
       }
 
     expandAnswers(qaIndex) {
         const qas = [...this.state.qas];
```

Comment:
can you just write const qas = this.state.qas? 
is it necessary to spread an array into an array here?